ocs_ci/ocs/resources/pvc.py

```diff
@@ -74,6 +74,28 @@ def backed_pv_obj(self):
         pv_obj.reload()
         return pv_obj
 
+    def delete(self, wait=True, force=False, validate_pv_deleted=False):
+        """
+        Delete the PVC
+
+        Args:
+            wait (bool): True for wait for PVC deletion, False otherwise
+            force (bool): True for force deletion with --grace-period=0,
+                False otherwise
+            validate_pv_deleted (bool): True for respective PV deletion
+                validation, False otherwise
+
+        Returns:
+            dict: Dictionary represents a returned yaml file
```

Comment:
NIT: Leave empty line

Reply:
Could you check the return type. looks like we are getting str

Reply:
> NIT: Leave empty line

Where?


> Could you check the return type. looks like we are getting str

yaml.safe_load in ocs_ci.ocs.ocp.OCP#exec_oc_cmd returns a dict

> /home/ebenahar/git/ocs-ci/ocs_ci/ocs/ocp.py(80)exec_oc_cmd()
     79             from ipdb import set_trace; set_trace()
---> 80             return yaml.safe_load(out)
     81         return out

ipdb> type(yaml.safe_load(out))                                                                                                                                                                                    
<class 'dict'>

Reply:
> > NIT: Leave empty line
> 
> Where?
> 
        Returns:
            dict: Dictionary represents a returned yaml file

        """

> > Could you check the return type. looks like we are getting str
> 
> yaml.safe_load in ocs_ci.ocs.ocp.OCP#exec_oc_cmd returns a dict
> 
> > /home/ebenahar/git/ocs-ci/ocs_ci/ocs/ocp.py(80)exec_oc_cmd()
> > 79             from ipdb import set_trace; set_trace()
> > ---> 80             return yaml.safe_load(out)
> > 81         return out
> 
> ipdb> type(yaml.safe_load(out))
> <class 'dict'>

```
>>> pvc_obj = helpers.create_pvc(sc_name=sc_name, size='1Gi')
>>> pvc_obj.name
'pvc-test-2912182753'
>>> pvc_obj.status
'Bound'
>>> 
>>> 
>>> result = pvc_obj.delete()
>>> type(result)
<class 'str'>
>>> result
'persistentvolumeclaim "pvc-test-2912182753" deleted'
>>> 
```

Reply:
Fixed